pkg/manager/member/tikv_failover.go

```diff
@@ -30,6 +31,13 @@ func NewTiKVFailover(tikvFailoverPeriod time.Duration) Failover {
 }
 
 func (tf *tikvFailover) Failover(tc *v1alpha1.TidbCluster) error {
+	ns := tc.GetNamespace()
+	tcName := tc.GetName()
+	if len(tc.Status.TiKV.FailureStores) >= int(tc.Spec.TiKV.MaxFailoverCount) {
```

Comment:
IIUC, If multiple tikv-servers become `Down` simultaneously, the `MaxFailoverCount` may be exceeded.

Reply:
Good catch!

Addressed PTAL again.